plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -125,6 +130,15 @@ protected TestTable createTableWithUnsupportedColumn()
 
     // TODO move common tests from connector-specific classes here
 
+    @Override
+    public void testDelete()
+    {
+        // This test is covered by BaseConnectorTest.
+        // testDelete is very broad test that tests all kinds of delete operation,
+        // it assumes that either connector supports delete fully or not at all.
+        throw new SkipException("This connector supports metadata delete only");
```

Comment:
Don't skip the test. Instead, add proper conditions to the test (based on the declared `hasBehavior`)